packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -117,6 +134,17 @@ const shouldShowCounter = computed(() => props.withCounter && props.maxlength);
 
     <div v-if="props.message || shouldShowCounter" class="onyx-input__footer onyx-text--small">
       <span v-if="props.message" class="onyx-truncation-ellipsis">{{ props.message }}</span>
+      <OnyxTooltip
+        v-if="props.messageTooltip"
+        class="onyx-input__info-message"
+        open="hover"
+        position="bottom"
+        :text="props.messageTooltip"
+      >
+        <button :aria-label="t('infoTooltip')" class="onyx-input__tooltip-trigger">
+          <OnyxIcon :icon="circleInformation" color="neutral" size="12px" />
```

Comment:
```suggestion
          <OnyxIcon :icon="circleInformation" size="12px" />
```

Does not seem to have an effect since by default is inherits the font color which is already given by the label

Reply:
Same for the tooltip above

Reply:
just checked it, It has an effect